grpc-api-client/src/test/java/io/deephaven/grpc_api_client/barrage/chunk/BarrageColumnRoundTripTest.java

```diff
@@ -419,7 +419,7 @@ public void assertExpected(final WritableChunk<Attributes.Values> untypedOrigina
                 final ArrayList<ChunkInputStreamGenerator.FieldNodeInfo> fieldNodes = new ArrayList<>();
                 column.visitFieldNodes((numElements, nullCount) -> fieldNodes.add(new ChunkInputStreamGenerator.FieldNodeInfo(numElements, nullCount)));
                 final ArrayList<ChunkInputStreamGenerator.BufferInfo> bufferNodes = new ArrayList<>();
-                column.visitBuffers((offset, length) -> bufferNodes.add(new ChunkInputStreamGenerator.BufferInfo(offset, length)));
+                column.visitBuffers((length) -> bufferNodes.add(new ChunkInputStreamGenerator.BufferInfo(length)));
```

Comment:
this seems silly if BufferInfo were replaced with Long - TLongCollection or something if we don't want boxed values...? it should just be `column.visitBuffers(bufferNodes::add)` or so, right?